app/app.js

```diff
@@ -6,13 +6,15 @@ define([
   'structure/structureModule',
   'contacts/contactsModule',
   'login/loginModule',
+  'authentication/authenticationModule',
 ], function(angular) {
 
   return angular.module('portfolio', [
     'ui.router',
     'portfolio.structure',
     'portfolio.contacts',
-    'portfolio.login'
+    'portfolio.login',
+    'portfolio.authentication'
```

Comment:
Should the authentication module be a dependancy of the login module? Or do we need it elsewhere too?